docs/api-docs/management-apis/tutorials/contextual-filters-tutorial.md

```diff
@@ -0,0 +1,307 @@
+
+### Limitations
+* Currently, the API only supports the default Channel with an ID of 1.
+* You can configure contextual filters for Category pages, but not for Brand or Search Results pages.
+
+###  Prerequisites
+
+* [A BigCommerce store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial).
+* API `access_token` with `content modify` scope.
+* Knowledge of the [Settings API](https://developer.bigcommerce.com/api-reference/store-management/settings).
+* Pro or Enterprise store plan. To upgrade, see [Changing Your Store’s Plan](https://support.bigcommerce.com/s/article/How-do-I-upgrade-my-stores-plan).
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> * Your store plan must support Product Filtering to use contextual filters. Product Filtering is available for Pro and Enterprise plans. To upgrade, see [Changing Your Store’s Plan](https://support.bigcommerce.com/s/article/How-do-I-upgrade-my-stores-plan).
+> * Enterprise accounts have the ability to use [custom fields](https://support.bigcommerce.com/s/article/Custom-Fields) as product filters to provide shoppers with additional ways to customize their search. 
+
+</div>
+</div>
+</div>
+
+## View pre-configured filters
+
+There are two ways to retrieve the global filters pre-configured on your store: using the Settings API or through the store’s control panel. To retrieve all pre-configured global filters using the API, send a `GET` request to [/v3/settings/search/filters](insert-link-here).
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate#requestrunner) // replace link
+
+The response will contain all of the global filters currently configured on your store. 
+
+```json
+{
+   "data": [
+       {
+           "id": "Y2F0ZWdvcnk=",
+           "display_name": "Category",
+           "type": "category",
+           "items_to_show": 15,
+           "collapsed_by_default": false,
+           "display_product_count": true,
+           "is_enabled": true
+       },
+       {
+           "id": "U2l6ZQ==",
+           "display_name": "Size",
+           "type": "product",
+           "display_product_count": true,
+           "collapsed_by_default": false,
+           "items_to_show": 10,
+           "sort_by": "alpha",
+           "is_enabled": true
+       },
+       {
+           "id": "cHJpY2U=",
+           "display_name": "Price",
+           "type": "price",
+           "collapsed_by_default": false,
+           "is_enabled": true
+       }
+       ...
+   ],
+   "meta": {}
+}
+```
+The global filters returned in the response of the `GET` request should match the filters you see in the control panel under **Storefront Filters**. This set of filters is what you currently have configured on your store. 
```

Comment:
The global filters returned from completing a `GET` request should match the filters you see in the control panel under **Storefront Filters**. This set of filters is what you currently have configured on your store.